akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -754,7 +754,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
 
   def receiveGossipStatus(status: GossipStatus): Unit = {
     val from = status.from
-    if (!latestGossip.overview.reachability.isReachable(selfUniqueAddress, from))
+    if (!latestGossip.isReachable(selfUniqueAddress, from))
```

Comment:
does this filter correctly?
e.g. `{n1, n2} | {n3, n4}`
n1 marked n4 as unreachable
n3 gossips to n4 - is that ignored? it should be influenced by a observation by n1